test_conformance/c11_atomics/common.h

```diff
@@ -1053,15 +1053,15 @@ CBasicTest<HostAtomicType, HostDataType>::KernelCode(cl_uint maxNumDestItems)
                 finalDest[dstItemIdx] =
                     atomic_flag_test_and_set_explicit(destMemory+dstItemIdx,
                                                       memory_order_relaxed,
-                                                      memory_scope_work_group);)";
+                                                      memory_scope_device);)";
```

Comment:
It looks like it is possible to reach this line for the `LocalMemory()` path.  That means `destMemory` could be `__local`.  While `memory_scope_device` is allowed for local memory accesses it provides no additional guarantees over `memory_scope_work_group` for that case.

If it avoids need to skip this configuration for the local memory case then this value could be changed based on `LocalMemory()`.